packages/cardano-services/environments/.schedule.preview.json

```diff
@@ -0,0 +1,10 @@
+{
+  "schedules": {
+    "list": [
+      {
+        "queue": "pool-delist-schedule",
+        "cron": "0 * * * *"
```

Comment:
Is there a reason to have a schedule config per network? They all seem to be the same

Reply:
For the given task of delisting, they are the same. But they might become different in time, and there will be for sure more schedules that we can't anticipate right now.
What I can do is, I can step down to a single file and once we need more, we can have more.

Reply:
Let's consider https://github.com/input-output-hk/cardano-js-sdk/pull/950#discussion_r1360698371 first, if we decide to keep the files then there's no need to downgrade